kilkari/src/main/java/org/motechproject/nms/kilkari/domain/Subscriber.java

```diff
@@ -170,9 +170,9 @@ public int hashCode() {
     @Override
     public String toString() {
         return "Subscriber{" +
-                "callingNumber=" + callingNumber +
-                ", dateOfBirth=" + dateOfBirth +
-                ", lastMenstrualPeriod=" + lastMenstrualPeriod +
+                "callingNumber=" + "***" +
+                ", dateOfBirth=" + "***" +
+                ", lastMenstrualPeriod=" + "***" +
```

Comment:
Instead of replacing with "***" I would just remove those fields.  I would also log id so the record can be found in the database if we need to debug something related to it.